### ISSUE TITLE
508-defect-1: Form Confirmation page focus is lost

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#59616]
### Point of contact

@jeana-adhoc

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

When navigating from the review/submit application page to the confirmation page, focus is lost. The desired element is not read out by a screen reader.

Attached are 3 videos from chrome, safari, and firefox on a mac. In the videos I show navigating from the submit page to the confirmation page. I also show using `document.activeElement` to have the console show me what is the active focus on the page. I also reload the confirmation page to show where focus goes. And in all three cases there is no focus

https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/8c70c783-af63-4247-9e43-459f690ea05e

https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/827fcad3-ab85-4c75-af60-4ed22b7e778e

https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/60bb1561-82ab-4c8b-b8bf-ad03bd8ea6af

### Reproduction steps

1. I've tested this on forms 21-4142, 26-4555, and 20-0995 on staging with the same results.
  - Here is the [confirmation page for 21-4142](https://staging.va.gov/supporting-forms-for-claims/release-records-to-va-form-21-4142/confirmation)
2. Start the form, navigate to the review page
3. Turn on a screen reader (Same results in Mac Voiceover and Jaws)
4. Submit the form.
5. Usually Nothing is announced.
  -  In one instance in Jaws, Windows, Chrome, the `<title>` tag was read by the screen reader
  - In another instance, when reloading the confirmation page in safari and typing `document.activeElement` in the console you can see that the desired `<h2>` does receive focus but is still not announced.

In some cases, while watching the console, I can see the desired `<h2> Thank you for submitting your request</h2>` receive focus but then it quickly goes to the `<body>`

### What I expect

When navigating to the confirmation page, I expect the `<h2>Thank you for submitting your authorization request</h2>` to be announced by the screen reader and receive focus.

### Proposed solution or next steps

This seems like a complicated engineering task because it is so inconsistent.

### References, articles, or WCAG support

1. [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [X] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by Mottie** [comment]
We fixed this in Supplemental Claims by adding a [delayed focus](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/appeals/995/containers/ConfirmationPage.jsx#L26)

```js
scrollTo('topScrollElement');
// make sure you're not targeting a print-only H2
waitForRenderThenFocus('h2');
```

**Comment by jeana-adhoc** [comment]
This little bit from @Mottie was actually very helpful.  I was able to make the fix here in this PR: https://github.com/department-of-veterans-affairs/vets-website/pull/24425


https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/e7e3327f-5e3a-4707-b9d8-fab4c2dde4c0


https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/b25ab1b6-ad03-4fb1-84a4-39130e6f64c3


https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/6c0946a8-bf80-4942-baca-88706221cc7d
